docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc

```diff
@@ -24,7 +24,7 @@ spec:
           - sh
           - -c
           - |
-            bin/elasticsearch-plugin install --batch analysis-icu
+            bin/elasticsearch-plugin remove --purge analysis-icu ; bin/elasticsearch-plugin install --batch analysis-icu
```

Comment:
Not tested but I think we can take advantage of multiline with `|`:
```suggestion
            bin/elasticsearch-plugin remove --purge analysis-icu
            bin/elasticsearch-plugin install --batch analysis-icu
```

Reply:
Absolutely, it works as well.
However, I had the feeling that explicitely showing the `;` (which escapes the error code for the first command) was a bit more explicit (comparing to usual `&&` which fails after first command fails)

Reply:
Should be fixed with new commit

Reply:
> However, I had the feeling that explicitely showing the ; (which escapes the error code for the first command) was a bit more explicit (comparing to usual && which fails after first command fails)

Yes-ish, I'm ok with `cmd1; cmd2` but this is equivalent to declare the 2 commands on 2 lines.

Reply:
Definitely. And it keeps the line length below the 80 chars limit as well 👍